Jobba.Cron/Implementations/CronService.cs

```diff
@@ -12,18 +12,16 @@ private static CronExpression GetCronExpression(string expression)
         => CronExpression.Parse(expression, CronFormat.IncludeSeconds);
 
     public DateTimeOffset? GetNextExecutionDate(string expression)
-        => GetCronExpression(expression).GetNextOccurrence(DateTimeOffset.UtcNow, TimeZoneInfo.Utc, true);
+        => GetNextExecutionDate(expression, DateTimeOffset.UtcNow);
```

Comment:
maybe we should keep the next execution time in a dictionary so we can make sure we don't miss a job if the timer fails to run every second?